docs/install/conda.rst

```diff
@@ -17,7 +17,7 @@ Binary Package (For Users)
 Binary distributions of the latest release for mac and linux (64-bit) 
 using the conda package manager can be installed by running the command::
 
-    conda install -c cyclus -c pyne pyne=VERSION
+    conda install -c conda-forge -c pyne pyne=VERSION
```

Comment:
I would recommend changing these lines to 

```
conda install -c conda-forge pyne
```

The PyNE channel is out of date and no longer in use.